src/routes/v1/token_test.go

```diff
@@ -0,0 +1,45 @@
+package v1_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/dana-team/platform-backend/src/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateServiceAccount(t *testing.T) {
+	serviceAccount := types.ServiceAccount{
+		ServiceAccountName: "new-sa",
+	}
+	body, _ := json.Marshal(serviceAccount)
+	request, _ := http.NewRequest("POST", "/v1/namespaces/default/serviceaccounts/", bytes.NewBuffer(body))
+	request.Header.Set("Content-Type", "application/json")
+
+	writer := httptest.NewRecorder()
+	router.ServeHTTP(writer, request)
+
+	assert.Equal(t, http.StatusOK, writer.Code)
+	var response types.ServiceAccount
+	err := json.Unmarshal(writer.Body.Bytes(), &response)
+	assert.NoError(t, err)
+	assert.Equal(t, "new-sa", response.ServiceAccountName)
+}
+
+func TestGetToken(t *testing.T) {
+	serviceAccount := types.ServiceAccount{
+		ServiceAccountName: "test-sa",
+	}
+	body, _ := json.Marshal(serviceAccount)
+	request, _ := http.NewRequest("GET", "/v1/namespaces/test-namespace/token/", bytes.NewBuffer(body))
+	writer := httptest.NewRecorder()
+	router.ServeHTTP(writer, request)
+
+	assert.Equal(t, http.StatusOK, writer.Code)
```

Comment:
It returns not found here because it couldn't find the created service account in the tests.

@mzeevi @fran2810 Do you have idea why? I'll debug it later.

Reply:
Aren't you creating the `serviceAccount` in the `default` namespace and then asking for a token in the `test-namespace` namespace?

Reply:
Nope this is not the issue. I had to make a mock of creating token.